src/esm_runscripts/prepexp.py

```diff
@@ -358,22 +359,23 @@ def update_runscript(fromdir, scriptsdir, tfile, gconfig, file_type):
                     f"Original {file_type} different from target",
                     differences
                     + "\n"
-                    + "Note: You can choose to use -U flag in the esm_runscripts call "
-                    + "to automatically update the runscript (WARNING: This "
-                    + f"will overwrite your {file_type} in the experiment folder!)\n",
+                    + "Note: You can choose to use ``-U`` flag in the "
+                    + "``esm_runscripts`` call to automatically update the runscript "
+                    + f"(WARNING: This will overwrite your {file_type} in the "
+                    + "experiment folder!)\n",
                 )
-                correct_input = False
-                while not correct_input:
-                    update_choice = questionary.confirm(
-                        f"Do you want that {scriptsdir + '/' + tfile} is "
-                        + "updated with the above changes?"
-                    ).ask()
-                    if update_choice:
-                        correct_input = True
-                        oldscript = fromdir + "/" + tfile
-                        print(oldscript)
-                        shutil.copy2(oldscript, scriptsdir)
-                        print(f"{scriptsdir + '/' + tfile} updated!")
+                update_choice = questionary.confirm(
+                    f"Do you want that {scriptsdir}/{tfile} is "
+                    + "updated with the above changes?"
+                ).ask()
+                if update_choice:
+                    oldscript = fromdir + "/" + tfile
+                    print(oldscript)
+                    shutil.copy2(oldscript, scriptsdir)
+                    print(f"{scriptsdir + '/' + tfile} updated!")
+                elif not update_choice:
+                    print("Submission stopped")
+                    sys.exit(0)
```

Comment:
I would exit with 1 here instead of 0. exit code 1 is the standard "something went wrong" message, and I would say that this counts at "something went wrong"

Reply:
Do you really think that is should be 1. Isn't that the user's choice?

Reply:
that turns more and more into a philosophical question, one in which I need more coffee to address. In the end, I would say `esm_runscripts` should return `0` if the run ends up on the batch system, and `1` (or something more specific) if that is not the case